tests/src/test/scala/common/Wsk.scala

```diff
@@ -73,6 +75,21 @@ case class WskProps(
     def overrides = Seq("-i", "--apihost", apihost, "--apiversion", apiversion)
 }
 
+case class WskPropsV2(
```

Comment:
A scala `case` cannot extend from another scala `case` class.  When the V1 interface is removed, I think a single WskProps case class can be had.